src/Microsoft.Performance.Toolkit.Engine.Tests/DataSourceSetTests.cs

```diff
@@ -0,0 +1,581 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Microsoft.Performance.SDK.Processing;
+using Microsoft.Performance.Testing;
+using Microsoft.Performance.Testing.SDK;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source123;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source4;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Performance.Toolkit.Engine.Tests
+{
+    [TestClass]
+    public class DataSourceSetTests
+        : EngineFixture
+    {
+        private DataSourceSet Sut { get; set; }
```

Comment:
What does `Sut` mean? Can we rename this to something more descriptive?

Reply:
Sut means "System Under Test." It is the thing that is being tested.

Reply:
cool. I didn't know that. Thanks.